packages/applications/ssr/src/components/pages/représentant-légal/_utils/steps/SaisieType.step.tsx

```diff
@@ -43,31 +54,40 @@ export const SaisieTypeStep: FC<SaisieTypeStepProps> = ({
       />
     ))
     .with('personne-morale', () => (
-      <Situation
-        contexte={contexte}
-        nom="Une personne morale"
-        informationÀRemplir="le nom de la société"
-        piècesJustificatives={
-          <>
-            <li>
-              Pour les sociétés constituées :
-              <ul className="mt-2 ml-4 list-disc">
-                <li>un extrait Kbis</li>
-              </ul>
-            </li>
-            <li>
-              Pour les sociétés en cours de constitution :
+      <>
+        <SaisieTypeSociétéStep
+          onChange={(typeSociété) => {
+            onChange &&
+              onChange({
+                type: stateTypeReprésentantLégal,
+                typeSociété,
+              });
+            setStateTypeSociété(typeSociété);
+          }}
+        />
+        <Situation
+          contexte={contexte}
+          nom="Une personne morale"
+          informationÀRemplir="le nom de la société"
+          piècesJustificatives={match<TypeSociété>(stateTypeSociété)
+            .with('constituée', () => <li>un extrait Kbis</li>)
+            .with('en cours de constitution', () => (
               <ul className="mt-2 ml-4 list-disc">
                 <li>une copie des statuts de la société</li>
                 <li>
                   une attestation de récépissé de dépôt de fonds pour constitution de capital social
                 </li>
                 <li>une copie de l’acte désignant le représentant légal de la société</li>
               </ul>
-            </li>
-          </>
-        }
-      />
+            ))
+            .with('non renseignée', () => (
+              <li className="italic">
+                Veuillez choisir le type de société pour voir les pièces justificatives à fournir
```

Comment:
```suggestion
                Veuillez sélectionner le type de société pour voir les pièces justificatives à fournir
```
Pour cohérence de wording